common/lib/xmodule/xmodule/templates/problem/multiplechoice.yaml

```diff
@@ -2,40 +2,53 @@
 metadata:
```

Comment:
@ahsan-ul-haq, apply my comment above to this multiple choice question type as well.